### PR TITLE
Clarify prerequisites for Windows Container Workloads

### PR DESCRIPTION
**Pull request by LorbusChris**
/cc @aravindhp 

Preview: https://deploy-preview-39585--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/enabling-windows-container-workloads

### CODE REVIEW

**Review by aravindhp — COMMENTED** [review]
Thanks for fixing this, @LorbusChris 
/lgtm
/cc @codyhoag

File: windows_containers/enabling-windows-container-workloads.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Before adding Windows workloads to your cluster, you must install the Windows Ma
 
 * You have installed the OpenShift CLI (`oc`).
 
-* You have installed your cluster using installer-provisioned infrastructure. Clusters installed with user-provisioned infrastructure are not supported for Windows container workloads.
+* You have installed your cluster using installer-provisioned infrastructure, or using user-provisioned infrastructure with a `platform: none` setting.
```
Comment (codyhoag) [inline windows_containers/enabling-windows-container-workloads.adoc:17]:
```suggestion
* You have installed your cluster using installer-provisioned infrastructure, or using user-provisioned infrastructure with the `platform: none` field set in your `install-config.yaml` file.
```

**Review by codyhoag — COMMENTED** [review]
Thanks for catching this, @LorbusChris! Just one comment on slightly more detail around that `platform: none` field.

**Comment by codyhoag** [comment]
LGTM!

**Pull request merged**